src/Kronecker.jl

```diff
@@ -2,10 +2,10 @@ module Kronecker
 
 export GeneralizedKroneckerProduct, AbstractKroneckerProduct, AbstractSquareKronecker
 export AbstractKroneckerSum, KroneckerSum
-export SquareKroneckerProduct, EigenKroneckerProduct, ShiftedKroneckerProduct
+export EigenKroneckerProduct, ShiftedKroneckerProduct
 export issquare, getmatrices, size, getindices, order, issymmetric, isposdef
 export ⊗, kronecker, Matrix, ⊕, kroneckersum
-export tr, det, logdet, collect, inv, +, *, mult!, eigen, /, adjoint, transpose, conj, solve, exp
+export tr, det, logdet, collect, inv, +, *, mult!, eigen, /, adjoint, transpose, conj, solve, exp, kron
```

Comment:
Don't think that `mult!` exists and more, so can remove from this list.

Reply:
Is that suppose to be `mul!`?

Reply:
There used to be a `mult!` function that this package implemented, but a previous PR changed it to `mul!`. In either case, we probably shouldn't be exporting it.

Reply:
Sure about this? I think it might be useful to export the inplace multiplication? Definitely should rename to `mul!` though.

Reply:
`LinearAlgebra` doesn't do it, and we're extending their method, so I don't think that we should.